src/Uno.Foundation.Runtime.Wasm/Interop/JSObject.wasm.cs

```diff
@@ -20,7 +19,6 @@ public sealed class JSObject
 		/// Used by javascript to dispatch a method call to the managed object at <paramref name="handlePtr"/>.
 		/// </summary>
 		[Obfuscation(Feature = "renaming", Exclude = true)]
-		[Preserve]
 		public static void Dispatch(string handlePtr, string method, string parameters)
 		{
 			var intPtr = _strToIntPtr(handlePtr);
```

Comment:
Codacy found an issue: [Use the overload that takes a 'CultureInfo' or 'IFormatProvider' parameter.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5872302)